### ISSUE TITLE
bit remote bit remote add ssh://root@0.0.0.0:32771:/tmp/scope -g     fail

### ISSUE THREAD

**Issue by dehuiZhang** [teambit/bit-docker#3]
![image](https://user-images.githubusercontent.com/21165726/81767854-fffa6400-950b-11ea-98b5-f0187d99109b.png)

ssh root@0.0.0.0 -p 32771   is  ok
bit remote need username and password？？ why？

**Comment by marmxlab** [comment]
I ran into the same issue yesterday and then I tried the solution on the other issue https://github.com/teambit/bit-docker/issues/2 and it worked for me.

Instead of running the image on docker hub, try run the image that you built at the first step on the instruction. (The instructions on this repo seem to be incorrect)

Don't do this after building your image
`docker run --rm --name bit -d -P  --volume ~/.ssh/id_rsa.pub:/tmp/id_rsa.pub bitteam/bit-docker`

Do this instead
`docker run --rm --name bit -d -P  --volume ~/.ssh/id_rsa.pub:/tmp/id_rsa.pub bit`

Then run `docker port bit 22` to see which port it has been mapped to and use it in your `bit remote add` command.
